src/main/java/com/wavefront/config/GeneratorConfig.java

```diff
@@ -73,6 +77,10 @@ public void initPropertiesFromFile() throws IOException {
     // if traceTypesCount is set, it has precedence so the traces will be generated with default
     // parameters
     traceTypesCount = rootNode.path("traceTypesCount").asInt(0);
+    totalTraceCount = rootNode.path("totalTraceCount").asInt(0);
+    if (totalTraceCount < 0) {
+      totalTraceCount = 0;
+    }
```

Comment:
why we need to default it to 0, as later you check if totalTraceCount > 0

Reply:
Agree, fixed.